packages/providers/src.ts/base-provider.ts

```diff
@@ -760,14 +761,24 @@ export class BaseProvider extends Provider {
             if (data) {
                 const filtered = data.transactions.filter((e: { result: string; }) => e.result != 'DUPLICATE_TRANSACTION');
                 if (filtered.length > 0) {
-                    const contractsEndpoint = MIRROR_NODE_CONTRACTS_ENDPOINT + transactionId;
-                    const dataWithLogs = await axios.get(this._mirrorNodeUrl + contractsEndpoint);
-                    const record = {
+                    let record: any;
+                    record = {
```

Comment:
I think that for `CRYPTOCREATE` transactions we can populate:

-  `from` -> based on the transaction ID (the sender of the tx)
-  `timestamp` -> the consensus timestamp of the TX

Reply:
Done